platform/Tiltfile

```diff
@@ -1,5 +1,3 @@
-allow_k8s_contexts(['orbstack', 'docker-desktop'])
```

Comment:
please keep this here. without this if your current local `kubectl` context is some remote cluster (ex. GKE), you could potentially accidentally tear-down resources there